plugins/chat/assets/javascripts/discourse/controllers/chat-channel-edit-title.js

```diff
@@ -27,11 +27,12 @@ export default class ChatChannelEditTitleController extends Controller.extend(
 
   @action
   onSaveChatChannelTitle() {
-    return ChatApi.modifyChatChannel(this.model.id, {
-      name: this.editedTitle,
-    })
-      .then((chatChannel) => {
-        this.model.set("title", chatChannel.title);
+    return this.chatApi
```

Comment:
Not essential at all, but it might be nice to move towards async/await eventually

e.g.

```javascript
@action
async onSaveChatChannelTitle() {
  try {
    const { channel } = await this.chatApi.updateChannel(this.model.id, {
      name: this.editedTitle,
    });
    this.model.set("title", channel.title);
    this.send("closeModal");
  } catch (event) {
    if (event.jqXHR?.responseJSON?.errors) {
      this.flash(event.jqXHR.responseJSON.errors.join("\n"), "error");
    }
    // Should probably add handling for other types of errors as well?
  }
}

Reply:
Yes I need to start using it more, I kinda hate the try/catch mechanism but this is what it is.

Reply:
I have taken note of this, will investigate using more async/await after the merge :+1: